src/pandablocks_ioc/ioc.py

```diff
@@ -1824,6 +1861,18 @@ def create_block_records(
 
         return record_dict
 
+    def create_version_records(self, fw_vers_dict: dict[str, str]):
+        """Creates handful of records for tracking versions of IOC/Firmware via EPICS
+
+        Args:
+            fw_vers_dict (dict[str, str]): Dictionary mapping firmwares to versions
+        """
+
+        builder.stringIn("VERSION", initial_value=__version__)
+        for item in fw_vers_dict:
+            basename = item.replace(" ", "_").upper()  # Normalize name
+            builder.stringIn(f"{basename}_VERSION", initial_value=fw_vers_dict[item])
```

Comment:
I think we should

`PANDA:VERSION:PVI`

and

```
PANDA:VERSION:IOC
PANDA:VERSIONS:PANDA_SW
PANDA:VERSIONS:FPGA
PANDA:VERSIONS:ROOTFS
```

Reply:
I would prefer if we put this in the `SYSTEM` block in a `VERSIONS` group, but apart from that I agree